easy_my_coop/models/coop.py

```diff
@@ -192,6 +192,10 @@ def _compute_subscription_amount(self):
                                        required=True,
                                        readonly=True,
                                        states={'draft': [('readonly', False)]})
+    share_product_currency_id = fields.Many2one(
+        related="share_product_id.currency_id",

```

Comment:
Currency should be retrieved by the company.

Reply:
here is an example https://github.com/coopiteasy/vertical-cooperative/blob/12.0/easy_my_coop_loan/models/loan.py#L63

Reply:
Ok, I will change that.